lib/src/mesh_network.dart

```diff
@@ -91,4 +92,7 @@ class MeshNetwork implements IMeshNetwork {
   @override
   Future<void> selectProvisioner(int provisionerIndex) =>
       _methodChannel.invokeMethod('selectProvisioner', {'provisionerIndex': provisionerIndex});
+
+  @override
+  Future<List<dynamic>> get provisionersUUIDList => _methodChannel.invokeMethod('getProvisionersUUID');
```

Comment:
Same here a type instead of dynamic

Reply:
same error here  'List dynamic ' is not a subtype of type 'List String ' error

Reply:
```dart
  @override
  Future<List<String>> get provisionersUUIDList async {
    final result = await _methodChannel.invokeMethod<List>('getProvisionersUUID');
    return result.cast<String>();
  }
```

then you will be able to use `List<String>`

Reply:
thanks